packages/repository/src/__tests__/fixtures/mixins/findByTitleRepoMixin.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Constructor} from '@loopback/context';
+import {CrudRepository, Model, Where} from '../../../index';
+import {FindByTitleInterface} from './findByTitleInterface';
```

Comment:
I think we can simply move `FindByTitleInterface` into this file. The interface name can be just `FindByTitle`.